version_bump.py

```diff
@@ -1,6 +1,6 @@
 # NEON AI (TM) SOFTWARE, Software Development Kit & Application Framework
 # All trademark and other rights reserved by their respective owners
-# Copyright 2008-2022 Neongecko.com Inc.
```

Comment:
This file can be removed; the shared workflow deprecated per-repository scripts like this